### DISCUSSION TITLE
Upper and lower case in the file name

### DISCUSSION THREAD

**Discussion by martin-arg** [remotely-save/remotely-save#487] [Q&A]
Hi I want to know, because I have problem with this before. Is the plugin compatible with filename in upper and lower case? I'm'using the onedrive personal sync at the moment, but I want to move to WebDAV. I use Obsidian in multiples system, so this is important to me.
Thanks

**Comment by fyears** [comment]
the plugin doesn't change the filename, and keep the upper or lower cases as is. 

the most important consideration is whether **your os** supports case-sensitive file names. 

if your system doesn't support case sensitive file names, and you change the files from AAA to aaa, the plugin will not work correctly deal to the os problem.
